target/linux/ramips/dts/mt7621_zbtlink_zbt-wg1602-v04.dtsi

```diff
@@ -72,15 +80,13 @@
 			gpio-export,output = <1>;
 			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
 		};
-
-		ext-usb {
-			gpio-export,name = "ext-usb";
-			gpio-export,output = <1>;
```

Comment:
We need to use `enable-active-high` and `GPIO_ACTIVE_HIGH` here. I did some tests about this:
```
	gpio-export,output	GPIO_ACTIVE_	output level
		0		LOW		low
		1		LOW		high
		0		HIGH		low
		1		HIGH		high
```

Reply:
Fixed.